Source/MARKRangeSlider.m

```diff
@@ -1,4 +1,5 @@
 #import "MARKRangeSlider.h"
+#import "ADToolTipView.h"
```

Comment:
I guess it should be removed, ADToolTipView is never used and now it doesn't compile with it.